src/scripts/main.js

```diff
@@ -1,3 +1,52 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+const promiseHandler = () => {
+  const promise1 = new Promise((resolve) => {
+    resolve('Promise was resolved!');
+  });
+
+  promise1
+    .then((response) => {
+      const divBlock = document.createElement('div');
+
+      divBlock.classList.add('message');
```

Comment:
Move this logic for add class to the helper variable and use it everywhere, it will be better